src/pudl/etl/__init__.py

```diff
@@ -45,8 +45,15 @@
     *load_assets_from_modules([pudl.transform.ferc714], group_name="clean_ferc714"),
     *load_assets_from_modules([glue_assets], group_name="glue"),
     *load_assets_from_modules([static_assets], group_name="static"),
-    *load_assets_from_modules([pudl.output.denorm_eia], group_name="denorm_eia"),
-    *load_assets_from_modules([pudl.output.denorm_ferc1], group_name="denorm_ferc1"),
+    *load_assets_from_modules(
+        [
+            pudl.output.new_eia,
+            pudl.output.new_eia860,
+            pudl.output.new_eia923,
+        ],
+        group_name="new_eia",
+    ),
```

Comment:
There's a lot more going on in here than denormalization, with all the generation fuel stuff, and I think we're just going to swap in this module for the `eia923` output module that exists right now so `new` seemed more honest.